extension/src/popup/views/Account/hooks/useGetAccountData.tsx

```diff
@@ -216,24 +218,70 @@ function useGetAccountData(options: {
     const resolvedData = state.data;
 
     const interval = setInterval(async () => {
-      const publicKey = resolvedData.publicKey;
-      const networkDetails = resolvedData.networkDetails;
-      const balancesResult = await fetchBalances(
-        publicKey,
-        _isMainnet,
-        networkDetails,
-        false,
-      );
+      try {
```

Comment:
another small adjustment: let's fail quietly if we throw an error on update. A user's session should not crash after we've already loaded some data - let's let them continue with potentially stale data